CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project's packages adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## v0.1.3 
+
+### Added
+
+- Add giantswarm supporting option
```

Comment:
Maybe change this to add giantswarm monitoring option since you renamed it in the chart?